proto/frequenz/api/weather/weather.proto

```diff
@@ -74,14 +75,12 @@ message ForecastRequest {
   // will be streamed.
   repeated ForecastFeature features = 2;
 
+  // Optional for StreamLiveWeatherForecast.
```

Comment:
It would be nice if we could continue specifying what happens if this field is not specified.

Reply:
Agree, or actually what happens if it is provided in case of streaming. Is the field ignored (as suggested in the docs to streaming above) or will you initially receive historical data before the stream starts?

Reply:
> or will you initially receive historical data before the stream starts

I'd suggest to not support this right now.